credentialAtomicQuerySigOnChainSmt.go

```diff
@@ -0,0 +1,304 @@
+package circuits
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/big"
+	"strconv"
+
+	core "github.com/iden3/go-iden3-core"
+	"github.com/iden3/go-iden3-crypto/babyjub"
+	"github.com/iden3/go-merkletree-sql"
+	"github.com/pkg/errors"
+)
+
+// AtomicQuerySigInputs ZK private inputs for credentialAtomicQuerySig.circom
+type AtomicQuerySigOnChainSmtInputs struct {
+	BaseConfig
+
+	// Identity anti-track inputs
+	StateInOnChainSmtProof *merkletree.CircomVerifierProof
```

Comment:
Why not merkletree.Proof?

Reply:
Answered the same question in the **identity-server** repo.

Reply:
Will be fixed as discussed

Reply:
fixed